solutions/standard/DA-types.md

```diff
@@ -0,0 +1,101 @@
+# Configuring complex inputs in Event Notification
+
+Several optional input variables in the IBM Cloud [Event Notification deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure you deployable architecture.
```

Comment:
Was that hyperlink supposed to redirect to the actual tile? I'm not sure it makes sense to just link to the catalog page when you included the text "Event Notification" (which by the way is also missing the "s" at the end). 
If a user is reading this doc, they more than likely got here via the variable description, meaning they probably already added the DA to a project and are not configuring it.
@SirSpidey any thoughts?

Reply:
If users don't need to go to the DA from here, just remove the link, but use the correct name of the DA in the catalog. If you do keep the link, I agree with Conall that it should go to the "details" page of the DA in the catalog.

Reply:
Did some rewording